### ISSUE TITLE
Feature request: provide a webm version of animated gif posts

### ISSUE THREAD

**Issue by HigherFive** [r888888888/danbooru#2405]
Danbooru already does this for ugoira. I feel that the graphics interchange format falls under the same category of being so heavy that its file size limits its usability.

**Comment by ToksT** [comment]
I assume this would be for users with slow internet so they can load the image faster?

**Comment by HigherFive** [comment]
Not really, I think it would apply to most people. My connection is pretty fast, yet the playback of many gifs I look at is slowed down.

This is just something that I think would be neat to get, but the real issue is that there is no real control applied to gifs with a large bit rate. In comparison, images that are deemed too large receive samples and ugoira (as mentioned already) receive webms. I believe webm uploads get samples as well (if not, they should!).

**Comment by ToksT** [comment]
> I believe webm uploads get samples as well (if not, they should!).

They don't. Neither to swfs.

**Comment by HigherFive** [comment]
I must've been imagining things then. Or maybe it was their direct URL that made me think so.

**Comment by r888888888** [comment]
This should only be done for large gifs, say >1MB.
